pandas/util/_validators.py

```diff
@@ -518,3 +520,39 @@ def validate_insert_loc(loc: int, length: int) -> int:
     if not 0 <= loc <= length:
         raise IndexError(f"loc must be an integer between -{length} and {length}")
     return loc
+
+def validate_bool_kwargs_from_keywords(*keywords):
```

Comment:
umm why cannot we juse call the existng `validate_bool_kwarg`? why make yet another function?